### PR TITLE
Publish 1.13.15

### PR DESCRIPTION
**Pull request by Tatsinnit**
Publishing version 1.3.15, in this release we with add few key updates as described below. Thanks heaps and cc: @hsubramanianaks, @peterbom and, @qpetraroia ❤️🙏🎊🥷 fyi - @gambtho 

Following fixes/feature will be going out with this release:

* Avoid lengthy kubectl output changing the layout of containing elements.
* Handle explicit state for poller.
* Abort last operation on cluster.
* Remove the unsed method.
* Implement Create Cluster as WebView.
* Add state management.
* Update ASO version and move to webview-ui project.
* Fix comon id issue.
* Default branch is now main.
* Move kubectl commands to webview.

Thanks heaps.

### CODE REVIEW

File: docs/book/src/features/k8s-api-health-points.md [hunk 1]
```diff
@@ -2,8 +2,10 @@
 
 ### Run Kubernetes API Health Endpoints
 
-Right click on your AKS cluster and click on **Kubernetes API Health Endpoints** to easily run few [k8s cluster API endpoint health check](https://kubernetes.io/docs/reference/using-api/health-checks/). Currently we have enable following health check:
+Right click on your AKS cluster and click on **Run Kubectl Command** and as shown in the image below user can easily run health checks by selecting on in the new kubectl run command panel. Currently we have enable following health check:
```
Comment (peterbom) [inline docs/book/src/features/k8s-api-health-points.md:5]: A bit shorter:
```md
Right click on your AKS cluster and click on **Run Kubectl Commands**. Select and run health check commands from the *Health* section as shown in the image below. Currently we provide:
```

File: docs/book/src/features/run-kubectl-command.md [hunk 2]
```diff
@@ -9,4 +9,10 @@ Right click on your AKS cluster and click on **Run Kubectl Commands** to easily
 - API Resources
 - Get Cluster Info
 - Get Node
-- Get All Events
\ No newline at end of file
+- Get All Events
+
+User can also run the custom command by typing the kubectl command parameter in the text area and click `Run`.
```
Comment (peterbom) [inline docs/book/src/features/run-kubectl-command.md:14]: How about:
```md
User can also run custom commands by typing or editing `kubectl` command parameters in the text field. Custom commands can optionally be saved for future use.
```

**Review by peterbom — APPROVED** [review]
Looks great - thanks!

**Review by hsubramanianaks — APPROVED** [review]
looks good to me.

**Comment by qpetraroia** [comment]
Commands look great! However, there is a validation issue during the cluster create experience. With this PR, we would allow users to create clusters with non-valid syntax resulting in the user watching the "cluster creating" animation for what seems to be forever. Could we fix this with some validation?

image

**Comment by Tatsinnit** [comment]
> Commands look great! However, there is a validation issue during the cluster create experience. With this PR, we would allow users to create clusters with non-valid syntax resulting in the user watching the "cluster creating" animation for what seems to be forever. Could we fix this with some validation? image
> 
> example: image
> 
> However, it does look like we have some validation here: image

Thanks @qpetraroia or @hsubramanianaks and @peterbom  - can we log this as a bug against a repo and go ahead with this release how or shall we fix it, I do feel that is we have proper AKS level validation docs in a tracking bug we can do better fix for it. But just honestly leaving the floor open.

**Comment by qpetraroia** [comment]
With #263 added. We are good to merge! Thanks team @Tatsinnit, @peterbom, @hsubramanianaks!

**Review by qpetraroia — APPROVED** [review]

**Pull request merged**